src/Common/tests/System/PlatformDetection.cs

```diff
@@ -14,6 +14,7 @@ public static class PlatformDetection
     {
         public static bool IsWindows { get; } = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
         public static bool IsWindows7 { get; } = IsWindows && GetWindowsVersion() == 6 && GetWindowsMinorVersion() == 1;
+        public static bool IsWindows8 { get; } = IsWindows && GetWindowsVersion() == 6 && (GetWindowsMinorVersion() == 2 || GetWindowsMinorVersion() == 3);
```

Comment:
IsWindows8x sounds good to me. please update it and push it one more time. thanks.